plugins/libplugin-pay.c

```diff
@@ -386,6 +386,16 @@ void payment_start(struct payment *p)
 	payment_start_at_blockheight(p, INVALID_BLOCKHEIGHT);
 }
 
+static void channel_hint_to_json(const char *name, const struct channel_hint *hint, struct json_stream *dest)
```

Comment:
This would normally be called "json_add_channel_hint(struct json_stream *, const char *, const struct channel_hint *)".  It's internal, but consistency is nice.